webpack.mix.js

```diff
@@ -12,6 +12,5 @@ const mix = require('laravel-mix');
  */
 
 mix.js('resources/js/app.js', 'public/js')
-    .postCss('resources/css/app.css', 'public/css', [
-        //
-    ]);
+    .sass('resources/sass/app.scss', 'public/css');
```

Comment:
OMG was it that easy to set up sass? I love this mix thing so much...

Reply:
Yes. More magic. :sparkles: